### DISCUSSION TITLE
EmptyState fails to apply --loaded class

### DISCUSSION THREAD

**Discussion by james0r** [Shopify/polaris-react#11992] [Q&A]
Just following the Getting Started QRCode app tutorial. Upon loading my app in the shopify admin, the image loads but fails to be visible, lacking the loaded class with `opacity: 1` class. However, if I create a QRCode, then delete the QRCode and return to the app root route, the loaded class is applied and the image shows as expected.
<img width="690" alt="image" src="https://github.com/Shopify/polaris/assets/34731936/cbd5c92e-c5e3-4499-915e-2d7291d25184">

**Comment by boxedcode** [comment]
This also happens for me with the following dependency versions:

```
    "@shopify/polaris": "^13.9.0",
    "@shopify/shopify-app-remix": "^3.8.0",
```
